crypto/internal.h

```diff
@@ -125,6 +125,26 @@
 #if defined(_MSC_VER)
 #define alignas(x) __declspec(align(x))
 #define alignof __alignof
+#elif defined(__GNUC__) && \
+    (__GNUC__ * 10000 + __GNUC_MINOR__ * 100 + __GNUC_PATCHLEVEL__) < 40800
```

Comment:
This may have been noticed already. For clang-10 (pretending 4.2), https://github.com/awslabs/aws-lc/pull/162 art will be used here.